File: 260126/A부터 B까지 짝수의 합/sum-of-even-nums-from-a-to-b.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);
        int[] arr=new int[2];
        int answer=0;

        for(int i=0; i<2; i++){
            arr[i]=sc.nextInt();
        }

        for(int i=arr[0]; i<=arr[1]; i++){
            if(i%2==0){
                answer+=i;
            }
        }

        System.out.println(answer);
    }
}

File: 260224/A부터 B까지 짝수의 합/sum-of-even-nums-from-a-to-b.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a,b;

    cin>>a>>b;

    int sum=0;

    for(int i=a; i<b+1; i++){
        i%2==0?sum+=i:false;
    }

    cout<<sum;

    return 0;
}